File: NODE/models/employee.js
'use strict';
const { Model } = require('sequelize');

module.exports = (sequelize, DataTypes) => {
  class Employee extends Model {
    static associate(models) {
      // define association here
    }

    static addEmployee(emp) {
      return Employee.create(emp);
    }

    static getEmployees() {
      return Employee.findAll();
    }

    static getEmployeesPage(offset, pageSize) {
      return Employee.findAll({
        offset,
        limit: pageSize
      });
    }

    static getEmployeeById(id) {
      return Employee.findByPk(id);
    }

    static deleteEmployee(id) {
      return Employee.destroy({ where: { id } });
    }

    static updateEmployee(id, updatedFields) {
      return Employee.update(updatedFields, { where: { id } });
    }
  }

  Employee.init(
    {
      fullName: DataTypes.STRING,
      jobTitle: DataTypes.STRING,
      phoneNumber: DataTypes.STRING,
      email: DataTypes.STRING,
      address: DataTypes.STRING,
      city: DataTypes.STRING,
      state: DataTypes.STRING,
      primaryEmergencyContact: DataTypes.STRING,
      primaryEmergencyContactPhone: DataTypes.STRING,
      primaryEmergencyContactRelationship: DataTypes.STRING,
      secondaryEmergencyContact: DataTypes.STRING,
      secondaryEmergencyContactPhone: DataTypes.STRING,
      secondaryEmergencyContactRelationship: DataTypes.STRING,
    },
    {
      sequelize,
      modelName: 'Employee',
      underscored: false,
    }
  );

  return Employee;
};

File: Spring Boot JAVA/src/main/java/com/example/demo/entity/employee.java
package com.example.demo.entity;

import javax.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "employees_java")
public class employee {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(name = "fullName")
    private String fullName;

    @Column(name = "jobTitle")
    private String jobTitle;

    @Column(name = "phoneNumber")
    private String phoneNumber;

    @Column(name = "email")
    private String email;

    @Column(name = "address")
    private String address;

    @Column(name = "city")
    private String city;

    @Column(name = "state")
    private String state;

    @Column(name = "primaryEmergencyContact")
    private String primaryEmergencyContact;

    @Column(name = "primaryEmergencyContactPhone")
    private String primaryEmergencyContactPhone;

    @Column(name = "primaryEmergencyContactRelationship")
    private String primaryEmergencyContactRelationship;

    @Column(name = "secondaryEmergencyContact")
    private String secondaryEmergencyContact;

    @Column(name = "secondaryEmergencyContactPhone")
    private String secondaryEmergencyContactPhone;

    @Column(name = "secondaryEmergencyContactRelationship")
    private String secondaryEmergencyContactRelationship;
}
